introduction.md

```diff
@@ -0,0 +1,227 @@
+
+Over time, that began to change.  Much software became portable across
+architectures, which made it more valuable.  More to the point, it
+sometimes made one company's software valuable to its competitors.  To
+manage this situation, companies began to get serious about their
+copyrights.  Suddenly, the software that had been the subject of so
+much free (though clunky) collaboration became more difficult to work
+on because of copyright.
+
+In order to restore that previous collaborative dynamic, some
+developers began to share software under a new type of license that
+encouraged sharing instead of discouraging it.  Later, people wrote
+more licenses in the same vein.  They also developed technology and
+practices that made it easier to collaborate across distance,
+boundaries, and time.
+
+At some point, those collaborative practices and the licenses designed
+for sharing became known as "Free" or "Open Source" and more
+specifically identified as licenses that respect the [Open Source
+Definition](https://opensource.org/osd).  In the decades since,
+software made with these methods has taken over the world.  Every
+major consumer electronics product has at least some open source
+software inside it.  It is increasingly rare to find any electronic
+device of significant complexity that is wholly proprietary.
+
+As open open source software spreads through the technology world
+sector by sector, it has inspired open movements in adjacent fields
+and even distant domains.  In this set of modules, we will focus on
+open source software and related "digital public goods".  These are
+digital products that are freely shared in ways that invite
+permissionless copying and collaboration.
+
+The focus on DPGs necessarily excludes a variety of sectors that have
+also seen "open" progress.  Open hardware, open government, open
+science, and open organizations are all interesting applications of
+the original open source vision.  Any government agency interested in
+DPGs might also be interested in those topics.  They are not DPGs,
+though, so this set of modules will not cover them.  Instead, we focus
+on:
+
+ * Open Source Software - software that can be freely run, copied,
+   modified and distributed.
+
+ * Open Standards - these are standards that are freely available to
+   the general public.  Anybody can read them.  Typically, open
+   standards are made in ways that try to reduce or eliminate patent
+   license fees for implementors.
+
+ * Open Content - this is content in any media that may to some degree
+   be freely copied, displayed, modified or distributed, much like
+   open source software.  The most popular open content licenses are
+   the Creative Commons licenses.
+
+ * Open Data - these are datasets that may be freely copied and
+   shared.
+
+ * Open AI Models - these are machine learning models that may be
+   freely shared and are to some degree trained on open data sets and
```

Comment:
remove "to some degree"; if these conditions are not met, they are not open AI models.